src/conn/conn_las.c

```diff
@@ -0,0 +1,400 @@
+/*-
+ * Copyright (c) 2014-2015 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
```

Comment:
Agreed, and done.

Shall I go ahead and move `conn_cache.c` and `conn_cache_pool.c` into the `cache/` directory as well?